src/wallet-instance-attestation/issuing.ts

```diff
@@ -8,9 +8,13 @@ import { WalletInstanceAttestationIssuingError } from "../utils/errors";
 
 export class Issuing {
   walletProviderBaseUrl: string;
-
-  constructor(walletProviderBaseUrl: string) {
+  appFetch: GlobalFetch;
+  constructor(
+    walletProviderBaseUrl: string,
+    appFetch: GlobalFetch = { fetch }
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** Plain args:
 ‏
```suggestion 
    appFetch: GlobalFetch["fetch"] =  fetch
```

Reply:
(and of course it requires `this.appFetch.fetch` to become `this.fetch`)

Reply:
Thanks, I'll fix it